Automattic/vip-cli
# Force the preference for WebSocket in case we see an error during connection

## Description

Clients with multiple outbound IPs are experiencing connection issues since the sticky sessions are being [configured by IP](https://socket.io/docs/v3/using-multiple-nodes/#nginx-configuration).

This PR fixes this issue by setting the transport preference for WebSocket in case we see an error during the connection phase.

In case we don't see any connection errors, we keep the default transport order, [following the official documentation](https://socket.io/docs/v3/how-it-works/#upgrade-mechanism).

## Steps to Test

1. Check out PR.
2. Run `npm run build`
3. Run `./dist/bin/vip-wp.js @id.production --yes -- user list`
4. Command should execute successfully
